Websites/webkit.org/demos/html-switch/index.html

```diff
@@ -0,0 +1,359 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Switch control examples</title>
+    <link rel="stylesheet" href="demo.css">
+</head>
+<body>
+    <h1>Switch demos</h1>
+    <p>The markup for switches is <code>&lt;input type="checkbox" switch&gt;</code>. It comes built-in with <code>::thumb</code> and <code>::track</code> pseudo-elements. The <code>::before</code> and <code>::after</code> pseudo-elements can also be used for extra possibilities!</p>
+
+    <p>The switch control is enabled in <a href="https://developer.apple.com/safari/resources/">Safari Technology Preview</a>.</p>
+
+    <section class="example">
+        <h1>Native switch (unstyled)</h1>
+        <p>The native switch supports the <code>accent-color</code> and <code>color-scheme</code> CSS properties.</p>
+        <input type="checkbox" switch>
+    </section>
+
+    <div id="warning">
+        <p><b>Your browser does not support the <code>::thumb</code> and <code>::track</code> pseudo-elements.</b> The demos below will not render properly.</p>
+        <p>The pseudo-elements can be enabled in Safari Technology Preview behind the relevant feature flag in <em>Develop > Feature Flags</em>. See <a href="https://developer.apple.com/documentation/safari-developer-tools/develop-menu">Apple's documentation</a> on how to enable the develop menu if you don't yet have it enabled.</p>
+    </div>
+
+    <section class="example">
+        <h1>Basic styled switch</h1>
+        <input type="checkbox" switch class="basic-switch">
+        <details>
+            <summary>Show CSS</summary>
+            <style>
+.basic-switch {
+    appearance: none;
+    position: relative;
+    background-color: #98989d;
+    height: 20px;
+    width: 44px;
+    border-radius: 20px;
+}
+
+.basic-switch:checked {
+    background-color: #32d74b;
+}
+
+.basic-switch::thumb {
+    background-color: white;
+    width: 24px;
+    height: 24px;
+    margin: -2px -1px;
+    transition: all 0.3s;
+    border-radius: 100%;
+    box-shadow: 0 0 5px rgba(0,0,0,0.3)
+}
+
+.basic-switch:checked::thumb {
+    translate: 22px 0;
+}
+            </style>
+        </details>
+    </section>
+    <section class="example">
+        <h1>Basic styled switch with symbol</h1>
+        <input type="checkbox" switch class="basic-with-symbol">
+        <details>
+            <summary>Show CSS</summary>
+            <style>
+.basic-with-symbol {
+    appearance: none;
+    position: relative;
+    background-color: #98989d;
+    height: 20px;
+    width: 44px;
+    border-radius: 20px;
+}
+
+.basic-with-symbol:checked {
+    background-color: #007aff;
+}
+
+.basic-with-symbol::thumb {
+    background-color: white;
+    width: 24px;
+    height: 24px;
+    margin: -2px -1px;
+    transition: all 0.3s;
+    border-radius: 100%;
+    box-shadow: 0 0 5px rgba(0,0,0,0.3)
+}
+
+.basic-with-symbol:checked::thumb {
+    translate: 22px 0;
+}
+
+.basic-with-symbol::before {
+    content: "〇";
```

Comment:
It's probably worth changing this too `content: "〇" / ""` making use of WebKit's new support for the content alt syntax, this improves the experience for screen reader users.

Reply:
It hasn't landed in STP despite what we claimed in the last release notes, but yeah I agree otherwise